app/services/hyrax/visibility_propagator.rb

```diff
@@ -10,7 +10,7 @@ class VisibilityPropagator
     # @return [#propagate]
     def self.for(source:)
       case source
-      when Hyrax::WorkBehavior # ActiveFedora
+      when ActiveFedora::Base # ActiveFedora
```

Comment:
Question: I missed it, where did this change?

Reply:
The reason is that the monograph https://github.com/samvera/hyrax/blob/main/.dassie/app/models/monograph.rb model won't include Hyrax::WorkBehavior, which will cause tests related to visibility changes failed for fileset visibility propagating.

Reply:
ah, the generated model doesn't include `WorkBehavior`. it seems like the weird thing here is that the `VisibilityCopyJob` is casting the valkyrie resource back to an AF::Base, even when `use_valkyrie` is `true`(?)

i think in the ideal case, both of the following would be true:
  - `VisibilityCopyJob` should support works generated by Wings; being generous about the type check as proposed here seems good.
  - the `use_valkyrie` code path here would leverage valkyrie objects throughout.

i think it would be worth checking up about the second point as part of this work.